x-pack/winlogbeat/module/security/config/winlogbeat-security.js

```diff
@@ -1242,55 +1417,115 @@ var security = (function () {
         evt.Put("winlog.logon.failure.sub_status", descriptiveFailureStatus);
     };
 
-    var addUACDescription = function (evt) {
+    var addUACDescription = function(evt) {
         var code = evt.Get("winlog.event_data.NewUacValue");
         if (!code) {
             return;
         }
-        var uacCode = parseInt(code);
-        if (isNaN(uacCode)) {
-            return;
-        }
-        var uacResult = [];
-        for (var i = 0; i < uacFlags.length; i++) {
-            if ((uacCode | uacFlags[i][0]) === uacCode) {
-                uacResult.push(uacFlags[i][1]);
+        var uac_code=parseInt(code);
+        var uac_result = [];
+        for (var i=0; i<uac_flags.length; i++) {
+            if ((uac_code | uac_flags[i][0]) === uac_code) {
+                uac_result.push(uac_flags[i][1]);
             }
         }
-        if (uacResult.length > 0) {
-            evt.Put("winlog.event_data.NewUacList", uacResult);
+        if (uac_result) {
+            evt.Put("winlog.event_data.NewUACList",uac_result);
         }
+        var uac_list=evt.Get("winlog.event_data.UserAccountControl").replace(/\s/g,'').split("%%").filter(String);
+        if (! uac_list) {
+            return;
+        }
+        evt.Put("winlog.event_data.UserAccountControl",uac_list);
```

Comment:
```suggestion
        evt.Put("winlog.event_data.UserAccountControl", uac_list);
```

For formatting consistency and readability it would be good to consitently use a space between args.